### PR TITLE
improve arch/nodes.md info on Conditions and Allocatable

### PR DESCRIPTION
**Pull request by jaypipes**
Adds information about capacity versus allocatable in the `architecture/nodes.md` doc as well as clarifies some of the information about `Node.Status.Conditions`.

### CODE REVIEW

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ architecture design doc for more details.
 A node's status contains the following information:
 
 * [Addresses](#addresses)
-* [Condition](#condition)
-* [Capacity](#capacity)
+* [Conditions](#conditions)
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md]: This change will break inbound hyperlinks.
Instead, you could combine
```suggestion
 * [Conditions](#condition)
```

with the suggestion below.

File: content/en/docs/concepts/architecture/nodes.md [hunk 2]
```diff
@@ -41,9 +41,9 @@ The usage of these fields varies depending on your cloud provider or bare metal
 * InternalIP: Typically the IP address of the node that is routable only within the cluster.
 
 
-### Condition
+### Conditions
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md]:
```suggestion
### Conditions {#condition}
```

File: content/en/docs/concepts/architecture/nodes.md [hunk 3]
```diff
@@ -88,11 +92,19 @@ Enabling this feature creates a small delay between the
 time when a condition is observed and when a taint is created. This delay is usually less than one second, but it can increase the number of Pods that are successfully scheduled but rejected by the kubelet.
 {{< /caution >}}
 
-### Capacity
+### Capacity and Allocatable
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md]:
```suggestion
### Capacity and Allocatable {#capacity}
```
would keep the hyperlink for this section be the same, ie https://kubernetes.io/docs/concepts/architecture/nodes/#capacity

**Comment by jaypipes** [comment]
> Some (optional) tips on how to preserve inbound hyperlinks.
> 
> This is especially important for very popular pages, it's also relevant to less popular documents where particular sections get linked to from other sites.

Ah, great suggestions, thank you Tim! I'll update and push up a new PR shortly.

Best,
-jay

**Comment by jaypipes** [comment]
OK @sftim all fixed up now. Thanks again.

**Pull request merged**